src/services/tweets-getter.service.ts

```diff
@@ -27,7 +27,12 @@ export const tweetsGetterService = async (twitterClient: Scraper): Promise<Tweet
     const tweets: Tweet[] = [];
     const tweetsIds = twitterClient.getTweets(TWITTER_HANDLE, 200);
 
+    let hasRateLimit = true;
     for await(const tweet of tweetsIds) {
+        const rateLimitTimeout = setTimeout(() => hasRateLimit = true,1000 * API_RATE_LIMIT);
```

Comment:
`hasRateLimit` seem never to be put to false?

Reply:
OMG I let it in debug mode. Thanks for catching it